test/test_database.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+from corpus import Corpus
+import os
+
+
+class TestDatabase(unittest.TestCase):
+    def setUp(self):
+        self.corpus = Corpus()
+
+    def test_messagefile(self):
```

Comment:
What is this test actually trying to verify? It doesn't actually invoke anything from alot.